packages/lit-dev-content/site/docs/ssr/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Server-side rendering (SSR)
+eleventyNavigation:
+  key: 🧪 Overview
+  parent: Server-side rendering
+  order: 1
+---
+
+<details class="pre-release">
+  <summary> 🧪 Pre-release software: not for production.</summary>
+  
+  Lit labs packages are published to get feedback from the wider community. This code shouldn't be used in production, but we encourage you to try it out and [file issues](https://github.com/lit/lit/issues/new/choose) for any bugs you find. For general feedback, please use the GitHub [discussion](https://github.com/lit/lit/discussions).
+
+  For more information about the Lit labs process, see [Lib Labs](/docs/libraries/labs/)
+
+</details>
+
+Server-side rendering (SSR) can help make pages work without JavaScript, either for robustness or performance reasons, or to work with web crawlers and bots that don't run a full browser. Some sites can render faster if they render static markup first, and optionally hydrate the components afterwards by loading the JavaScript.
+
+[Lit SSR](https://github.com/lit/lit/tree/main/packages/labs/ssr#readme) lets you render Lit components and templates to static HTML markup in non-browser JavaScript environments like Node. It works without fully emulating the browser's DOM, and takes advantage of Lit's declarative template format to support streaming.
```

Comment:
A suggestion for expanding the intro a bit:

```suggestion
Server-side rendering (SSR) is a technique for generating and serving the HTML of your components, including shadow DOM and styles, before their JavaScript implementations have loaded and executed. The two main reasons to consider SSR are:

- Faster first paint, so that your users see your page sooner.
- Improved indexing by bots that don't run JavaScript.

For a deeper dive into server-side rendering concepts and techniques generally, see [Rendering on the Web](https://web.dev/rendering-on-the-web/) on web.dev.

Lit supports server-side rendering through the [Lit SSR](https://github.com/lit/lit/tree/main/packages/labs/ssr#readme)  package. Lit SSR renders Lit components and templates to static HTML markup in non-browser JavaScript environments like Node. It works without fully emulating the browser's DOM, and takes advantage of Lit's declarative template format to support streaming.
```